pkg/cmd/server/origin/controller.go

```diff
@@ -148,5 +193,34 @@ func (c *MasterConfig) NewOpenshiftControllerInitializers() (map[string]controll
 	}
 	ret["openshift.io/service-serving-cert"] = serviceServingCert.RunController
 
+	sdnController := controller.SDNControllerConfig{
+		NetworkConfig: c.Options.NetworkConfig,
+	}
+	ret["openshift.io/sdn"] = sdnController.RunController
+
+	ret["openshift.io/resource-quota-manager"] = controller.RunResourceQuotaManager
+
+	clusterQuotaReconciliationController := controller.ClusterQuotaReconciliationControllerConfig{
+		Mapper:                         c.ClusterQuotaMappingController.GetClusterQuotaMapper(),
+		DefaultResyncPeriod:            5 * time.Minute,
+		DefaultReplenishmentSyncPeriod: 12 * time.Hour,
+	}
+	ret["openshift.io/cluster-quota-reconciliation"] = clusterQuotaReconciliationController.RunController
+
+	clusterQuotaMappingController := controller.ClusterQuotaMappingControllerConfig{
+		ClusterQuotaMappingController: c.ClusterQuotaMappingController,
+	}
+	ret["openshift.io/cluster-quota-mapping"] = clusterQuotaMappingController.RunController
```

Comment:
@deads2k spawned https://github.com/openshift/origin/issues/14698 to track this and eventually fix this.